src/cs/lib/msquic_generated.cs

```diff
@@ -213,11 +215,11 @@ public unsafe partial struct QUIC_CREDENTIAL_CONFIG
 
         public QUIC_CREDENTIAL_FLAGS Flags;
 
-        [NativeTypeName("QUIC_CREDENTIAL_CONFIG::(anonymous union at src/inc/msquic.h:274:5)")]
+        [NativeTypeName("QUIC_CREDENTIAL_CONFIG::(anonymous union at src/inc/msquic.h:279:5)")]
```

Comment:
a bit annoying that it embeds line numbers.

Reply:
Yeah. I plan on looking into some changes here (Because I also have to post process to remove the full path). But thats out of scope for sure.